pkg/resource/rest.go

```diff
@@ -54,7 +54,29 @@ func GetAuthorizedRestConfig(apiServer, apiServerToken, caData string, tls *rest
 	return
 }
 
-func BuildRestConfig(apiServer, apiServerToken, caData string, tls *rest.TLSClientConfig) (*rest.Config, error) {
+func GetAuthorizedRestConfigFromFiles(apiServer, apiServerTokenFile, caFile string, tls *rest.TLSClientConfig,
```

Comment:
The duplication is inelegant, I’ll rework this.

Reply:
I think you reworked this in your last update? Marked as resolved.